docset/windows/bitlocker/Remove-BitLockerKeyProtector.md

```diff
@@ -42,22 +42,37 @@ Any encrypted data on the drive remains encrypted.
 
 We recommend you have at least one recovery password as key protector to a volume in case you need to recover a system.
 
-For an overview of BitLocker, see [BitLocker Drive Encryption Overview](https://technet.microsoft.com/en-us/library/cc732774.aspx) on TechNet.
+For an overview of BitLocker, see [Overview of BitLocker Device Encryption](https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-device-encryption-overview-windows-10).
 
 ## EXAMPLES
 
-### Example 1: Remove a key protector for a volume
-```
+### Example 1: Remove the second key protector for a volume
+```powershell
 PS C:\> $BLV = Get-BitLockerVolume -MountPoint "C:"
-PS C:\> Remove-BitlockerKeyProtector -MountPoint "C:" -KeyProtectorId $BLV.KeyProtector[1].KeyProtectorId
+PS C:\> Remove-BitLockerKeyProtector -MountPoint "C:" -KeyProtectorId $BLV.KeyProtector[1].KeyProtectorId
 ```
 
 This example removes a key protector for a specified BitLocker volume.
```

Comment:
Can you expand on the explanation here, i.e. what is "[1]". Admins may not understand that this is calling on the second item in the 0-based index. It may even be "better" to extract the second key protector to a variable by filtering by some other value in the $BLV variable, i.e. `$kp = $BLV.KeyProtector | ?{$_.propertyName -eq "propertyValue"}`.

Reply:
The example actually says it removes the second key protector without specifying anything about the type. It was created like that in the begging, I just corrected the case in cmdlet. Because of that I added the second example which actually filters by type like you suggested.